### ISSUE TITLE
You need configs!

### ISSUE THREAD

**Issue by bneil** [asciiu/scalatrex#1]
So im adding some :) PR should be in a few

**Comment by bneil** [comment]
https://github.com/asciiu/scalatrex/pull/2

**Comment by asciiu** [comment]
Yes, of course. This is a work in progress. I just finished implementing all the endpoints today. I see you must have been bored to stumble across this project! (∩｀-´)⊃━☆ﾟ.*･｡ﾟ

**Comment by bneil** [comment]
Just miss coding with my friend. Thought this would be a fun collab 👍
